.github/workflows/integration.yaml

```
@@ -0,0 +1,139 @@
+name: integration
+
+on:
+  push:
+    branches: [staging, stable]
+
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [staging]
+
+jobs:
+  libc-test:
+    name: libc-test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - plat: qemu
+          arch: x86_64
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Fetch helloworld
+      uses: actions/checkout@v3
+      with:
+        repository: unikraft/app-helloworld
+        fetch-depth: 1
+        path: _helloworld
+
+    - name: Build and run libc-test via uktest
+      uses: unikraft/kraftkit@staging
```

```suggestion
      uses: unikraft/kraftkit@v0
```

Why not like this and we do a tag?

---

Because this represents a branch or tag, there is no branch `v0` or tag `v0` on the kraftkit repo.

---

exactly, we could do it if you want ;)

A v0 branch wouldn't hurt, certainly easier to update than a tag

Should we do it?

---

I fear a `v0` branch is extra maintenance on our side. Once we do a proper release in kraftkit, we can put `unikraft/kraftkit@stable`

---

Okay 👍